config/env.sample

```diff
@@ -0,0 +1,5 @@
+PORT=5005
+CERT_FILE="config/public-cert.pem"
+KEY_FILE="config/private-key.pem"
+KEY_PASS=KEY_FILE
```

Comment:
Why would the password be the file path? A better default for `KEY_PASS` would rather be to just leave it out.